src/test/starknet.t.sol

```diff
@@ -122,8 +125,12 @@ contract StarknetTests is DssSpellTestBase, ConfigStarknet {
         assertEq(daiBridge.starkNet(), addr.addr("STARKNET_CORE"),   "StarknetTest/dai-bridge-core");
         assertEq(daiBridge.escrow(),   addr.addr("STARKNET_ESCROW"), "StarknetTest/dai-bridge-escrow");
 
-        assertEq(daiBridge.wards(addr.addr("MCD_PAUSE_PROXY")), 1, "StarknetTest/pause-proxy-not-ward-on-dai-bridge");
-        assertEq(daiBridge.wards(addr.addr("MCD_ESM")),         1, "StarknetTest/esm-not-ward-on-dai-bridge");
+        assertEq(daiBridge.wards(addr.addr("MCD_PAUSE_PROXY")),     1, "StarknetTest/pause-proxy-not-ward-on-dai-bridge");
+        assertEq(daiBridge.wards(addr.addr("MCD_ESM")),             1, "StarknetTest/esm-not-ward-on-dai-bridge");
+        // TODO verify that this is intentional or necessary
+        assertEq(daiBridge.wards(addr.addr("STARKNET_ESCROW_MOM")), 1, "StarknetTest/escrow-mom-not-ward-on-dai-bridge");
```

Comment:
This is not necessary.

Reply:
It currently is a ward of the new bridge but not the old bridge.  If it should not be a ward, we should remove that permission on the new bridge:
![image](https://user-images.githubusercontent.com/1408372/198099305-7cc26295-e746-43f0-b42e-d98aca24eaa3.png)

Reply:
Should we remove the auth in the spell?

Reply:
It seems not to be harmfull. It should be fixed If there is still time to fix it in this spell.